build/pipelines/templates/build-console-public.yaml

```diff
@@ -39,5 +39,4 @@ jobs:
     displayName: 'Publish XamlStyler.Console artifact'
     inputs:
       PathtoPublish: '$(Build.ArtifactStagingDirectory)'
-      ArtifactName: 'XamlStyler.Console'
-      publishLocation: 'Container'
\ No newline at end of file
+      ArtifactName: 'XamlStyler.Console'
```

Comment:
Hmm, I am seeing a new artifact now, but it appears to be the entire source tree + some additional folders. I _think_ we just want the nupkg output from 'a' sub-directory, right?

Reply:
just pushed a commit to fix that